### PR TITLE
Make the documentation on Search listener more obvious

### PR DESCRIPTION
**Pull request by simkimsia**

### CODE REVIEW

**Comment by simkimsia** [comment]
Meant to solve https://github.com/FriendsOfCake/crud/issues/328

File: docs/listeners/search.rst [hunk 1]
```diff
@@ -1,43 +1 @@
...
-
-    public $components = [
-      'Crud.Crud' => [
-        'actions' => ['index', 'view'],
-        'listeners' => ['Crud.Search']
-      ];
-
-  }
-  ?>
+Follow other .rst structure more closely
```
Comment (jippi) [inline docs/listeners/search.rst]: errr.. what?

**Comment by simkimsia** [comment]
Fixed the content for the search.rst

**Comment by simkimsia** [comment]
@jippi Is this okay for merge?

**Pull request merged**